lib/sles4sap.pm

```diff
@@ -347,26 +342,10 @@ sub copy_media {
     assert_script_run "mkdir $target";
     assert_script_run "mount -t $proto -o ro $path $mnt_path";
     $media_path = $mnt_path if script_run "[[ -d $media_path ]]";    # Check if specific ARCH subdir exists
-    assert_script_run "cp -ax $media_path/. $target/", $nettout;
+    assert_script_run "rsync --archive --checksum $media_path/. $target/", $nettout;
 
-    # Go back to target directory and umount the share, as we don't need it anymore
+    # Unmount the share, as we don't need it anymore
     assert_script_run "umount $mnt_path";
-
-    return 1 if get_var('DISABLE_CHECKSUM');
```

Comment:
now the checksum part is no more controllable by DISABLE_CHECKSUM...isn't it? Is it fine?

Reply:
yes... the checksum used by rsync is fast and lightweight so no need to have that variable anymore